.spelling

```diff
@@ -246,8 +246,6 @@ prepended
 retweets
 JetstackHQ
 acme-dns
-andreas-p
-renewBefore
```

Comment:
I think these two lines need to be added back for the verify script to pass.

Reply:
You are right, but the point of my PR was in fact to remove these two from `.spelling` 😅  The main issue with `.spelling` is that it does not get "used" by `./scripts/verify` since it clones the `release-next` branch without making sure .spelling is also grabbed.

Reply:
Lol my comment makes no sense 🤪

Reply:
why do you want to remove them from `.spelling`?

Reply:
Whatever the reason, I suppose you should then do it in a follow-up PR to this if this PR removes the _need_ for it?

Reply:
Good point, I'll move that `.spelling` trick to another PR